code/src/Components/TodoCounter.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+
+const TodoCounter = () => {
+
+    const todo = useSelector((store) => store.todo.items);
+    const remainingTodos = todo.filter((todo) => todo.done === false)
```

Comment:
Nice implementation of filtering out checked todo's to enable them to count as todos completed!